src/mflux_debugger/kill_all.py

```
@@ -0,0 +1,286 @@
+            if dry_run:
+                print(f"  [DRY RUN] Would kill: {get_process_info(proc)}")
+                killed_count += 1
+            else:
+                if force:
+                    proc.kill()  # SIGKILL
+                    print(f"  ☠️  Force killed: {get_process_info(proc)}")
+                else:
+                    proc.terminate()  # SIGTERM
+                    print(f"  ✅ Terminated: {get_process_info(proc)}")
+                killed_count += 1
+        except (psutil.NoSuchProcess, psutil.AccessDenied) as e:  # noqa: PERF203
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `PERF203`)

Remove unused `noqa` directive

(RUF100)